bochs/bx_debug/parser.y

```diff
@@ -459,6 +463,32 @@ symbol_command:
       }
     ;
 
+set_magic_break_points_command:
+      BX_TOKEN_SET_MAGIC_BREAK_POINTS '\n'
+      {
+        bx_dbg_set_magic_bp_mask(0);
```

Comment:
this will clear all magic breakpoints, is it expected ?
previously you had 'default' on BX

Reply:
yes, without args both commands clear all because who needs enable all? I documented this in doc and help, the BX thing for backward comp is **config** file

Reply:
![image](https://github.com/bochs-emu/Bochs/assets/9882181/ea4eaef2-f64a-4def-84aa-373c73deebf3)